src/ng-generate/prompter/prompts-questions/shared/prompt-complex-questions.ts

```diff
@@ -403,7 +403,7 @@ function getAllConstraints(allAnswers: any, templateHelper: TemplateHelper, answ
         : [];
     const allProperties = getAllPropertiesFromAspectOrEntity(templateHelper, selectedElement, allAnswers);
     const allowedPropertiesObjects = allProperties.filter(property => !excludedPropertiesUrns.includes(property.name));
-    const allowedProperties = allowedPropertiesObjects.map(property => loader.findByUrn(property.name) as Property);
+    const allowedProperties = allowedPropertiesObjects.map(property => loader.findByUrn(property.name) as Property).filter(item => item !== undefined);
```

Comment:
That seems a bit strange to me here. When I go through the test files we have nine "allowedProperitesObjects" after findByUrn has been executed only six are passed on ...

Remove the "filter" and look at the findByUrn - something is going wrong, or the Aspect Model Loader is not caching the elements correctly.

Reply:
i removed, there were undefined values because the model was invalid